src/quicklist.h

```diff
@@ -112,6 +114,7 @@ typedef struct quicklist {
     signed int fill : QL_FILL_BITS;       /* fill factor for individual nodes */
     unsigned int compress : QL_COMP_BITS; /* depth of end nodes not to compress;0=off */
     unsigned int bookmark_count: QL_BM_BITS;
+    unsigned int neg_fill_limit : QL_NEGATIVE_FILL_LIMIT_BITS; /* size limit of the quicklist node with negative 'fill' */
```

Comment:
just call it size_limit? and we can document that it's only valid when fill is negative.

Reply:
please also initialize it to -1 in quicklistCreate().